ShortestPath/ShortestPath/Program.cs

```diff
@@ -2,10 +2,11 @@
 
 namespace ShortestPath
 {
-    class program
+    class Program

```

Comment:
Why?

Reply:
Because best practice says that class are labeled PascalCase.